python/cudnn/sdpa/fwd/torch_op.py

```
@@ -0,0 +1,728 @@
+    SINKS = 4
+    SEQ_LEN_Q = 5
+    SEQ_LEN_KV = 6
+    RAGGED_Q = 7
+    RAGGED_KV = 8
+    RAGGED_O = 9
+    RAGGED_STATS = 10
+    RAGGED_V = 11
+    RAGGED_DQ = 12
+    RAGGED_DK = 13
+    RAGGED_DV = 14
+    O = 100
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)